app/engine/Rower.test.js

```diff
@@ -0,0 +1,517 @@
+  rower.handleRotationImpulse(0.010769)
+  rower.handleRotationImpulse(0.010707554)
+  rower.handleRotationImpulse(0.010722165)
+  rower.handleRotationImpulse(0.01089567)
+  rower.handleRotationImpulse(0.010917504)
+  rower.handleRotationImpulse(0.010997969)
+  rower.handleRotationImpulse(0.011004655)
+  rower.handleRotationImpulse(0.011013618)
+  rower.handleRotationImpulse(0.011058193)
+  rower.handleRotationImpulse(0.010807149)
+  rower.handleRotationImpulse(0.0110626)
+  rower.handleRotationImpulse(0.011090787)
+  rower.handleRotationImpulse(0.011099509)
+  rower.handleRotationImpulse(0.011131862)
+  rower.handleRotationImpulse(0.011209919)
+  testStrokeState(rower, 'Recovery')
+  testTotalMovingTimeSinceStart(rower, 0.9923165270000005)
+  testTotalLinearDistanceSinceStart(rower, 3.945590990721503)
+  testTotalNumberOfStrokes(rower, 3)
+  testCycleDuration(rower, 0.3712367640000004)
+  testCycleLinearDistance(rower, 1.1223674777910904)
+  testCycleLinearVelocity(rower, 4.469255430992754)
+  testCyclePower(rower, 249.9559970802513)
+  testDriveDuration(rower, 0.14348571700000012)
+  testDriveLinearDistance(rower, 0.6343816178819204)
+  testDriveLength(rower, 0.27227136331111523)
+  testDriveAverageHandleForce(rower, 177.6995123165446)
+  testDrivePeakHandleForce(rower, 294.67568796679427)
+  testRecoveryDuration(rower, 0.22775104700000026)
+  testRecoveryDragFactor(rower, 283.33086731525583)
+  testInstantHandlePower(rower, 0)
+  // Dwelling state starts here
+  rower.handleRotationImpulse(0.020769)
+  rower.handleRotationImpulse(0.020707554)
+  rower.handleRotationImpulse(0.020722165)
+  rower.handleRotationImpulse(0.02089567)
+  rower.handleRotationImpulse(0.020917504)
+  rower.handleRotationImpulse(0.020997969)
+  rower.handleRotationImpulse(0.021004655)
+  rower.handleRotationImpulse(0.021013618)
+  rower.handleRotationImpulse(0.021058193)
+  rower.handleRotationImpulse(0.020807149)
+  rower.handleRotationImpulse(0.0210626)
+  rower.handleRotationImpulse(0.021090787)
+  rower.handleRotationImpulse(0.021099509)
+  rower.handleRotationImpulse(0.021131862)
+  rower.handleRotationImpulse(0.021209919)
+  testStrokeState(rower, 'WaitingForDrive')
+  testTotalMovingTimeSinceStart(rower, 1.1025003730000005)
+  testTotalNumberOfStrokes(rower, 3)
+  testTotalLinearDistanceSinceStart(rower, 4.433576850630673)
+  testCycleDuration(rower, 0.36002684500000015)
+  testCycleLinearDistance(rower, 1.6103533377002601)
+  testCycleLinearVelocity(rower, 4.472870176389929)
+  testCyclePower(rower, 250.56298292575076)
+  testDriveDuration(rower, 0.14348571700000012)
+  testDriveLinearDistance(rower, 0.6343816178819204)
+  testDriveLength(rower, 0.27227136331111523)
+  testDriveAverageHandleForce(rower, 177.6995123165446)
+  testDrivePeakHandleForce(rower, 294.67568796679427)
+  testRecoveryDuration(rower, 0.21654112800000003)
+  testRecoveryDragFactor(rower, 283.33086731525583)
+  testInstantHandlePower(rower, 0)
+})
+
+// Test behaviour for noisy upgoing flank
```

Comment:
Are these placeholders for additional tests that we should add in the future? In that case add a `todo: add test case for ...` so we don't miss them when elaborating. If it means something else it would be good if you could make the comment more meaningful.